src/stores/config.store.ts

```diff
@@ -13,6 +16,20 @@ export const useThemeStore = defineStore(
       themes.value?.find(theme => theme.name === themeName.value)
     )
     const bgLayers = computed(() => config.value?.background_layers || [])
+    const layerTrees_3d = computed(() => {
+      const ol3d_groups = themes.value?.filter(
+        theme => theme.metadata?.ol3d_type !== undefined
```

Comment:
Should also test ` if (theme_ol3d_type !== 'terrain')` as in legacy.
Also, there is a layer missing (compared to the prod), entry is missing in fixtures and computed should return this missing layer.
![image](https://github.com/Geoportail-Luxembourg/luxembourg-geoportail/assets/115457104/14da4e99-3cf3-4b58-93bc-00e358ef57f9)

Reply:
@tkohr FYI update fixtures and translations in https://github.com/Geoportail-Luxembourg/luxembourg-geoportail/pull/80